packages/react-components/react-shared-contexts/src/CustomStyleHooksContext/CustomStyleHooksContext.ts

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable */
+
+import * as React from 'react';
+
+// The list of hooks is built by scraping the exports from react-components/src/index
+export type CustomStyleHooksContextValue = {
+  useAccordionHeaderStyles_unstable: (state: unknown) => void;
+  useAccordionItemStyles_unstable: (state: unknown) => void;
+  useAccordionPanelStyles_unstable: (state: unknown) => void;
+  useAccordionStyles_unstable: (state: unknown) => void;
+  useAvatarStyles_unstable: (state: unknown) => void;
+  useAvatarGroupStyles_unstable: (state: unknown) => void;
+  useAvatarGroupItemStyles_unstable: (state: unknown) => void;
+  useAvatarGroupPopoverStyles_unstable: (state: unknown) => void;
+  useBadgeStyles_unstable: (state: unknown) => void;
+  useCounterBadgeStyles_unstable: (state: unknown) => void;
```

Comment:
Aren't you missing `usePresenceBadgeStyles_unstable`?

```suggestion
  useCounterBadgeStyles_unstable: (state: unknown) => void;
  usePresenceBadgeStyles_unstable: (state: unknown) => void;
```